PRS1001-2000/PRS1376Abusaker.xml

```diff
@@ -32,25 +32,29 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          </xi:include>
       </encodingDesc>
       <profileDesc>
-         <langUsage><language ident="en">English</language></langUsage>
+         <langUsage><language ident="en">English</language><language ident="ar">Arabic</language><language ident="gez">Gǝʿǝz</language></langUsage>
       </profileDesc>
       <revisionDesc>
          <change who="PL" when="2016-04-28">batch updated according to requirements and issues.</change>
          <change who="PL" when="2016-03-21"> Created file from
                                                   google spreadsheet </change>
          <change who="ES" when="2016-02-09">CREATED: person</change>
+         <change when="2024-08-29" who="CH">Updated names and occupation; added wikidata.</change>
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPerson>
-            <person sex="1">
-               <persName>
-                  
-                  Abušakǝr
-               </persName>
-               <persName type="alt">Abū Šākir b. Abī l-Karam Buṭrus b. al-Muhaḏḏib</persName>
-               <occupation> wr. </occupation>
+            <person sex="1" sameAs="Q3147469">
+               <persName xml:lang="gez" xml:id="n1">አቡሻክር፡</persName>
+               <persName xml:lang="gez" corresp="#n1" type="normalized">ʾAbušākǝr</persName>
+               <persName xml:lang="ar" corresp="#n1">Abū Šākir ibn Abī l-Karam Buṭrus ibn al-Muhaḏḏib ibn al-Rāhib</persName>
```

Comment:
I would have it as n2, normal name should be preferably in Arabic script, transcription as normalized